.github/autobuild/mac.sh

```diff
@@ -71,12 +71,24 @@ setup() {
 }
 
 prepare_signing() {
+    ##  Certificate types in use:
+    # - MACOS_CERTIFICATE - Developer ID Application - for codesigning for adhoc release
+    # - MAC_STORE_APP_CERT - Mac App Distribution - codesigning for App Store submission
+    # - MAC_STORE_INST_CERT - Mac Installer Distribution - for signing installer pkg file for App Store submission
+
     [[ "${SIGN_IF_POSSIBLE:-0}" == "1" ]] || return 1
 
     # Signing was requested, now check all prerequisites:
     [[ -n "${MACOS_CERTIFICATE:-}" ]] || return 1
     [[ -n "${MACOS_CERTIFICATE_ID:-}" ]] || return 1
     [[ -n "${MACOS_CERTIFICATE_PWD:-}" ]] || return 1
+    [[ -n "${MAC_STORE_APP_CERT:-}" ]] || return 1
+    [[ -n "${MAC_STORE_APP_CERT_ID:-}" ]] || return 1
+    [[ -n "${MAC_STORE_APP_CERT_PWD:-}" ]] || return 1
+    [[ -n "${MAC_STORE_INST_CERT:-}" ]] || return 1
+    [[ -n "${MAC_STORE_INST_CERT_ID:-}" ]] || return 1
+    [[ -n "${MAC_STORE_INST_CERT_PWD:-}" ]] || return 1
+    [[ -n "${NOTARIZATION_PASSWORD:-}" ]] || return 1
     [[ -n "${KEYCHAIN_PASSWORD:-}" ]] || return 1
```

Comment:
This suggests that signing will be aborted if we do not also have certificates for app store submission. We need a way to allow just signing without store submission.